e2e/test/scenarios/actions/actions-in-object-detail-view.cy.spec.js

```diff
@@ -29,7 +29,9 @@ describe(
   { tags: ["@external", "@actions"] },
   () => {
     beforeEach(() => {
+      cy.intercept("GET", "/api/card/*").as("getModel");
```

Comment:
I assume this one isn't used anymore. So it should be deleted.

Reply:
383d7e5d3c031ebfc7a7c1b9db4542bfd815d596